tests/infra/consortium.py

```diff
@@ -711,10 +711,10 @@ def check_for_service(self, remote_node, status, recovery_count=None):
             current_status = r["service_status"]
             current_cert = r["service_certificate"]
             # Note: to change once this is backported to 2.x
-            if remote_node.version_after("ccf-2.0.3"):
-                current_recovery_count = r["recovery_count"]
-            else:
-                assert "recovery_count" not in r
+            # if remote_node.version_after("ccf-2.0.3"):
```

Comment:
See https://github.com/microsoft/CCF/pull/3994 for fix.

Reply:
Yes, waiting on that to merge.